linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/http/IstioIdentifier.scala

```diff
@@ -19,9 +19,15 @@ class IstioIdentifier(val pfx: Path, baseDtab: () => Dtab, routeCache: RouteCach
       case Some(host) =>
         Future.join(clusterCache.get(host), routeCache.getRules).map {
           case (Some(Cluster(dest, port)), rules) =>
-            val filteredRules = filterRules(rules, dest, req.headerMap.get)
+            //TODO: request scheme?
```

Comment:
I don't think this information is available on the HTTP request (unless it's in the request.uri).  We may have to punt on this.